Gems/ScriptCanvas/Code/Editor/View/Widgets/NodePalette/NodePaletteModel.cpp

```diff
@@ -1019,12 +1019,19 @@ namespace ScriptCanvasEditor
             GraphCanvas::TranslationRequestBus::BroadcastResult(catdetails, &GraphCanvas::TranslationRequests::GetDetails, catkey, catdetails);
 
             GraphCanvas::TranslationKey key;
-            key << "BehaviorClass" << methodClass.c_str() << "methods" << methodName.c_str() << "details";
+
+            AZStd::string updatedMethodName = methodName;
+            if (propertyStatus != ScriptCanvas::PropertyStatus::None)
+            {
+                updatedMethodName = (propertyStatus == ScriptCanvas::PropertyStatus::Getter) ? "Get" : "Set";
+                updatedMethodName.append(methodName);
+            }
```

Comment:
repeat: A string construction can be avoided by just appending the method name after the PropertyStatus if block

```suggestion
            AZStd::string updatedMethodName;
            if (propertyStatus != ScriptCanvas::PropertyStatus::None)
            {
                updatedMethodName = (propertyStatus == ScriptCanvas::PropertyStatus::Getter) ? "Get" : "Set";
            }

            updatedMethodName += methodName;
```